site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -303,9 +303,9 @@ public class MyAnnotatedService {
 
 </Tip>
 
-If multiple parameters exist with the same name in a query string, they can be injected as a `List<?>`
-or `Set<?>`, e.g. `/hello1?number=1&number=2&number=3`. You can use <type://@Default> annotation
-or `Optional<?>` class here, too.
+If the type is `List<?>` or `Set<?>`, multiple parameters exist with the same name in a query string can be
+injected, e.g. `/hello1?number=1&number=2&number=3`. Or you can use a comma-delimited parameter:
+`/hello1?number=1,2,3` You can use <type://@Default> annotation or `Optional<?>` class here, too.
```

Comment:
1) The sentence starts with `the type`, but it is unclear what type it is
2) The sentence starting with `Or` is not a complete sentence.

What do you think of the following?

```suggestion
If multiple parameters exist with the same name in a query string, they can be injected as a `List<?>`
or `Set<?>`, e.g. `/hello1?number=1&number=2&number=3`. If a single parameter is mapped to a
`List<?>` or `Set<?>`, it will be converted into comma-separated values, e.g. `/hello1?number=1,2,3`.
You can use <type://@Default> annotation or `Optional<?>` class here, too.
```